### PR TITLE
Add UI for user to enter its information

### PR DESCRIPTION
**Pull request by Jmion**
+ Add button to access it when registering is complete
WARNING: this is the interface ONLY as set in the sprint; the data is not collected YET

### CODE REVIEW

**Comment by Jmion** [comment]
This is a pull request that is resubmitted because the original one was mistakenly done onto master

**Comment by Jmion** [comment]
## Not pulling yet.

We are going to link it to the database to base a functional copy of this part of the program

**Comment by Jmion** [comment]
Going to merge to dev and then implement the data collection

**Comment by alexander-san** [comment]
Currently when an already registered E-Mail tries to register it gives a green checkmark but then says "Failed to register"

File: app/src/main/java/ch/epfl/sweng/favours/AuthentificationProcess.java [hunk 1]
```diff
@@ -57,6 +57,11 @@ public void afterTextChanged(Editable s) {
             isEmailCorrect.set(isEmailValid(binding.emailTextField.getText().toString()));
```
Comment (alexander-san) [inline app/src/main/java/ch/epfl/sweng/favours/AuthentificationProcess.java]: also when entering the E-Mail the first letter is capitalized which is not super bad as glitch but personally from the code in this PR I don't see why this happens. Maybe there is a field variable to turn this off.

**Comment by alexander-san** [comment]
@AndreaScalisi I added you to the assignees since you wanted to work on the bug fixing and removed you from the reviewers even though you did review the code [just for your record!]

**Comment by alexander-san** [comment]
@Charrlyine above you find my review.
Please take a look at the problems that arise from your confirmation mail implementation.
Good work (it works), just some issues remain to fix.
I think another intermediate screen is probably the easiest and nicest workaround

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -3,13 +3,14 @@
     package="ch.epfl.sweng.favours">
```
[the remark this answers is not in the document]
Comment (Jmion) [reply app/src/main/AndroidManifest.xml]: OK

File: app/src/main/java/ch/epfl/sweng/favours/AuthentificationProcess.java (hunk 1 above)
Comment (Jmion) [reply app/src/main/java/ch/epfl/sweng/favours/AuthentificationProcess.java]: Don't see the issue. Tried to reproduce it and failed.

**Comment by basbeu** [comment]
The issues reported by @alexander-san are going to be fixed on a different branch. Here the binding is working and as the integration test pass we merge it in the develop branch.

**Pull request merged**